packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx

```diff
@@ -481,11 +481,9 @@ describe("MafsGraph", () => {
             />,
```

Comment:
Something I should have done a long time ago - can you put a comment above line 471 saying "[0, 0] is the vertex"